docs/pages/9 - Contrib Modules.md

```diff
@@ -35,6 +35,43 @@ object project extends BuildInfo {
 
 * `def buildInfoPackageName: Option[String]`, default: `None`
   The package name of the object.
+  
+  
+### Flyway
+
+Enables you to configure and run [Flyway](https://flywaydb.org/) commands from your mill build file.
+The flyway module currently supports the most common flyway use cases with file based migrations.
+
+Configure flyway by overriding settings in your module. For example
+
+```scala
+object foo extends JavaModule with FlywayModule {
```

Comment:
Can you please add proper imports to the code snippet.

Reply:
For sure, it should get published as `mill.contrib.flyway.FlywayModule` right?

Reply:
Yes, that in line with the other contrib packages.